### ISSUE TITLE
Any sample for SSD1306 based OLED display?

### ISSUE THREAD

**Issue by pankaj-nikam** [raspberrypi/pico-examples#31]
Hi team, is there any sample for SSD1306 based OLED display? Can you please share? I am looking for 128x32 display.
Thanks.

**Comment by lurch** [comment]
There's an SSD1306 MicroPython example [here](https://github.com/raspberrypi/pico-micropython-examples/tree/master/i2c/1306oled) but I dunno how easy/hard it would be to convert to C. There's some C I2C examples [here](https://github.com/raspberrypi/pico-examples/tree/master/i2c).

**Comment by pankaj-nikam** [comment]
Yes, I found that sample and its working nice. I wanted C version because the Python currently does not support machine.RTC.

**Comment by HeadBoffin** [comment]
The Python example will be driven by C under the hood so you could fish it out of the Micropython source code

**Comment by mbober1** [comment]
I ported SSD1306 lib to RP2040. Feel free to use it.
https://github.com/mbober1/RPi-Pico-SSD1306-library

**Comment by pankaj-nikam** [comment]
Thanks for the porting @mbober1 😃

**Comment by lurch** [comment]
See also https://github.com/raspberrypi/pico-examples/pull/49 :slightly_smiling_face:

**Comment by lurch** [comment]
I guess this was "fixed" by #130 ? That got merged into https://github.com/raspberrypi/pico-examples/releases/tag/sdk-1.3.0 so take a look at https://github.com/raspberrypi/pico-examples/tree/master/i2c/oled_i2c

**Comment by mkerna** [comment]
There is an example in the repository at [.](https://github.com/raspberrypi/pico-examples/tree/master/i2c/ssd1306_i2c)  Please close this issue.
